### ISSUE TITLE
Balanced loss

### ISSUE THREAD

**Issue by gianmarco-terrones** [hspark1212/MOFTransformer#122]
Is there a way, for classification, to specify a class_weight of "balanced" similar to in sklearn? E.g. in sklearn.ensemble.RandomForestClassifier.

**Comment by hspark1212** [comment]
Thank you for your comment.

When it comes to classification in PyTorch, we use cross entropy loss as the default. In PyTorch, you can assign weights to the cross entropy loss. For more information, you can refer to the documentation of the CrossEntropyLoss class in PyTorch.
(https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html)

To assign weights to the classification in the PMTransformer, you can add an argument for weights to line 63 of the compute_classification function in the [objectives.py](https://github.com/hspark1212/MOFTransformer/blob/master/moftransformer/modules/objectives.py)  file.

**Comment by gianmarco-terrones** [comment]
Thanks!

Leaving a note here to myself that for Version 2.1.0, line 63 is `loss = F.cross_entropy(logits, labels)`
